src/cart/crt_internal_types.h

```diff
@@ -34,10 +34,12 @@ struct crt_grp_gdata;
 struct crt_na_config {
 	int32_t		 noc_port;
 	int		 noc_iface_total;
+	int               noc_domain_total;
```

Comment:
does not seem aligned

Reply:
> does not seem aligned

yeah, i think whole struct is badly aligned and clangformat is now forcing the right formatting on 1 field only:) will reformat next patch